pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -572,9 +572,8 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False, **
             all_params = 0
 
             for op_idx, tp in enumerate(trainable_params):
-                op, _ = tape.get_operation(
-                    op_idx
-                )  # get op_idx-th operator among differentiable operators
+                # get op_idx-th operator among differentiable operators
+                op, _, _ = tape.get_operation(op_idx, return_op_index=True)
```

Comment:
```suggestion
                op, _ = tape.get_operation(op_idx, return_op_index=False)
```
I know this change comes because `return_op_index=True`, will be the new default, but would it make sense to set it as `False`?